libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java

```diff
@@ -878,4 +896,13 @@ private Intent getServiceIntent() {
   private boolean isServiceAvailable() {
     return navigationService != null && isBound;
   }
+
+  private boolean checkValidLegIndex(int legIndex) {
```

Comment:
Minor comment - As this is checking if the leg index is _invalid_ (returns `true` if so), what about changing the name of the method to better reflect what's actually doing (e.g. `checkInvalidLegIndex`)?